src/btree/bt_sync.c

```diff
@@ -388,7 +388,7 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
 
     if (time_start != 0) {
         time_stop = __wt_clock(session);
-        __wt_verbose(session, WT_VERB_CHECKPOINT,
+        __wt_verbose_info(session, WT_VERB_CHECKPOINT,
```

Comment:
This message gets logged per file, I would suggest it to change to debug2.

Reply:
Fixed in fc9e331c099088bc387e439fe68fc54f24a4e2e6